packages/tools/examples/videoTools/index.ts

```diff
@@ -92,74 +117,357 @@ addButtonToToolbar({
   },
 });
 
+const toolsNames = [
+  LengthTool.toolName,
+  ProbeTool.toolName,
+  RectangleROITool.toolName,
+  EllipticalROITool.toolName,
+  CircleROITool.toolName,
+  BidirectionalTool.toolName,
+  AngleTool.toolName,
+  CobbAngleTool.toolName,
+  ArrowAnnotateTool.toolName,
+  PlanarFreehandROITool.toolName,
+  VideoRedactionTool.toolName,
+];
+let selectedToolName = toolsNames[0];
+
+addDropdownToToolbar({
+  options: { values: toolsNames, defaultValue: selectedToolName },
+  onSelectedValueChange: (newSelectedToolNameAsStringOrNumber) => {
+    const newSelectedToolName = String(newSelectedToolNameAsStringOrNumber);
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+
+    // Set the new tool active
+    toolGroup.setToolActive(newSelectedToolName, {
+      bindings: [
+        {
+          mouseButton: MouseBindings.Primary, // Left Click
+        },
+      ],
+    });
+
+    // Set the old tool passive
+    toolGroup.setToolPassive(selectedToolName);
+
+    selectedToolName = <string>newSelectedToolName;
+  },
+});
+
 addButtonToToolbar({
-  id: 'previous',
-  title: 'previous',
+  id: 'Next',
+  title: 'Next',
   onClick() {
-    viewport.scroll(-1);
+    navigateRelative(1);
   },
 });
```

Comment:
Please rename accordingly to avoid confusion. Something like "Select Next Annotation" and make sure the function name is related to selection.